.github/workflows/prbuild.yaml

```diff
@@ -7,7 +7,86 @@ env:
   GOPROXY: https://proxy.golang.org/
   SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
 jobs:
+  check-changelog:
```

Comment:
should we skip this whole job on `main` commits?

Reply:
Yes, we should.

Reply:
We actually have a separate build workflow for `main`, so I think we should just skip this workflow altogether for `main`. I've added this in.